tests/virt_autotest/kubevirt_tests_agent.pm

```diff
@@ -90,7 +90,7 @@ sub rke2_agent_setup {
     $self->check_service_status();
 
     # Start rke2-agent service ready
-    mutex_create('RKE2_AGENT_START_READY');
+    mutex_create('rke2_agent_start_ready');
```

Comment:
Does lower case make any difference ?

Reply:
Created lock name in server side is lower case, so just want to keep consistent with the server side.